packages/core/test/html-select/htmlSelectTests.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2018 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+import { assert } from "chai";
+import { EnzymePropSelector, mount, ReactWrapper } from "enzyme";
+import * as React from "react";
+
+import { HTMLSelect, IOptionProps } from "../../src/index";
+
+describe("<HtmlSelect>", () => {
+    const emptyHandler = () => {
+        return;
+    };
+
+    it("renders options without CSS classes by default", () => {
```

Comment:
i hate to say it, but... 🤷‍♂️ since you're adding a test suite here then might as well test that all the options props come out as expected, not just `className`.  keep it simple, short and sweet.